### PR TITLE
dnf: add examples

### PR DESCRIPTION
**Pull request by Iniesta8**
- [X] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [X] The page has 8 or fewer examples.
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [X] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).

### CODE REVIEW

**Comment by Iniesta8** [comment]
I've also removed the `sudo dnf install -y` example because it's trivial.

**Comment by sbrl** [comment]
I'm on the fence about the example removal there. On the one hand, it is trivial. Just because it's trivial doesn't mean to say that we should remove it! tldr-pages is aimed at beginners, and those to forget how a command works (amongst others).

On the other hand, assuming yes to all queries could be considered advanced and somewhat dangerous in nature. To that end, I'd prefer to remove it.

I'm not sure, what do you think?

/cc @agnivade

**Comment by einverne** [comment]
I've noticed that there is `install` and `install -y` in same page. How about merge that into one?

**Comment by Iniesta8** [comment]
The -y parameter also automatically installs dependencies of the desired package without the user having to confirm again. This can definitely lead to problems. In addition, the -y parameter of the install command is much less interesting than the other examples in the page.

**Comment by agnivade** [comment]
Given that the `yum` page has this, I think we should keep it since we have space for more examples.

**Comment by Iniesta8** [comment]
@agnivade : Ok, i've added the example with -y again. Ready to merge?

File: pages/linux/dnf.md [hunk 1]
```diff
@@ -17,3 +17,15 @@
...
+
+- List installed packages:
+
+`dnf list --installed`
+
+- Search packages via keyword:
+
+`sudo dnf search {{keyword}}`
+
+- Find what package provides the given value:
```
Comment (sbrl) [inline pages/linux/dnf.md]: Perhaps this will read better?
```suggestion
- Find which packages provide a given value:
```

**Pull request merged**